include/staff/ticket-tasks.inc.php

```diff
@@ -84,8 +85,10 @@ class="green button action-button ticket-task-action"
         $threadcount = $task->getThread() ?
             $task->getThread()->getNumEntries() : 0;
 
-        $viewhref = sprintf('#tickets/%d/tasks/%d/view',
-                $ticket->getId(), $id);
+        if ($access)
+            $viewhref = sprintf('#tickets/%d/tasks/%d/view', $ticket->getId(), $id);
+        else
+            $viewhref = '#';
```

Comment:
But it looks like `$viewhref` is not used in the following code (if `$access` is false)